packages/components/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
+    "strictBindCallApply": true,
```

Comment:
Can you explain me why you needed to add this option? 😄

Reply:
Yes, it's to ensure that `.bind({})` for functions enforce the typings.

Example here: https://github.com/drunkoders/cards/pull/125/files#diff-5a75b3b5d5cff2d3c58ea81dd1b63d9c86952e87c9575cd67352c3dfa1472659R5